design/MonitoringModeAPI.md

```diff
@@ -2974,71 +2974,165 @@ The response will contain a array of JSON object with the updated recommendation
             "container_image_name": "kruize/tfb-qrh:1.13.2.F_et17",
             "container_name": "tfb-server-1",
             "recommendations": {
+              "version": "1.0",
               "notifications": {
-                "112101": {
+                "111000": {
```

Comment:
I don't see notification codes doc with this code and a few other codes are missing, can you please check and update the doc.

Reply:
I think the doc in mvp_demo is updated, we will need to update the doc in the remote_monitoring branch